packages/drivers/http/src/http.js

```diff
@@ -54,15 +54,18 @@ const getDriver = (store) => {
           const { headers = {} } = options
           let appliedOptions = options
 
-          // attach data as JSON object
           if (data && ['object', 'array'].includes(typeof data)) {
-            if (!headers['Content-Type']) appliedOptions = { ...appliedOptions, headers: { ...headers, 'Content-Type': 'application/json' } }
-
-            appliedOptions = { ...appliedOptions, body: JSON.stringify(data) }
+            // attach data as JSON object
+            let body = data
+            if (!(data instanceof FormData)) {
+              headers['Content-Type'] = headers['Content-Type'] || 'application/json'
```

Comment:
Don't mutate parameters of a function.
Look at my first com' I used a new variable